MobileCenter/MobileCenter/Internals/Channel/MSChannelDefault.m

```diff
@@ -176,7 +176,7 @@ - (void)flushQueue {
                  // Forward logs to the sender.
                  [self.sender
                              sendAsync:container
-                     completionHandler:^(NSString *batchId, NSError *error, NSUInteger statusCode) {
+                     completionHandler:^(NSString *batchId, NSError *error, NSUInteger statusCode, NSData *data) {
```

Comment:
Feels weird to have the data param added as a forth param here and in the `sender:callCompletedWithStatus:data:error:` -callback it's added as a 3rd param.